app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :foods
+  has_many :recipes
+  validates :name, presence: true
+end
```

Comment:
it would be nice to add confirmable to your devise this is done so a user is able to confirm there email address after they signup.